Third-Party Notice - GitHub Actions Toolkit.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 GitHub
```

Comment:
Why is this needed?

Reply:
Because I directly lifted implementation details from their JavaScript code to make sure I got the escaping correct as it isn't documented and it actually varies based on which part of the workflow command you execute. So, to make sure I cover my bases, I included that code's license as a 3rd-party notice.

Reply:
Ah, I think I just read this
> permission notice shall be included in all copies or **substantial portions** of the Software

as a few mappings ported (not really copied) as non-substantial. But maybe you're right.

Either way, just by looking at the file name it's not very obvious what it relates to. I've seen some projects dedicating a separate folder (licensing?) for such things and mentioning the detail in README (maybe even README of that folder).

Reply:
I've expanded in the file a bit to explain it better.